packages/pancake-uikit/src/components/MenuItem/theme.ts

```diff
@@ -0,0 +1,22 @@
+import { darkColors, lightColors } from "../../theme/colors";
+
+export type MenuItemTheme = {
+  textColor: string;
+  activeTextColor: string;
+  backgroundColor: string;
+  borderColor: string;
+};
+
+export const light: MenuItemTheme = {
```

Comment:
Sorry if I'm being dim - not 100% on the purpose of these new theme objects? I think it's fine just to access `theme.colors.textSubtle/secondary/tertiary/primary` wherever these are used?

Reply:
I was under the impression that there were needed for light/dark mode. If not I'll gladly remove them.